fetch/fetch.go

```diff
@@ -11,24 +11,110 @@ import (
 )
 
 var (
+	trueBool = true
+)
+
+type Fetcher struct {
 	client *ssm.SSM
 	paths  []string
 	tags   []string
+	debug  bool
+}
 
-	trueBool = true
-)
+// MustSetEnv retrieves SSM parameters for the given paths and tags and sets each
+// as env variables via os.Setenv.  This function will panic if any error occurs.
+// Set debug to true for debug logging to STDOUT (WARNING: any sensitive SSM param values will be logged).
+func MustSetEnv(paths, tags []string, debug bool) {
```

Comment:
Renamed from `MustSetOS` so this is not backwards compatible.  We should create a new release anyways.